packages/plugin-awesome/package.json

```diff
@@ -27,7 +27,7 @@
     "clean": "rimraf ./dist",
     "eslint": "eslint ./src/**/*.{js,jsx,ts,tsx}",
     "eslint:format": "eslint --fix ./src/**/*.{js,jsx,ts,tsx}",
-    "test": "rimraf ./out && vitest run"
+    "test": "rimraf ./out && vitest run --config ./vitest.config.ts"
```

Comment:
Vitest resolves the config by default. We have a single configuration file, so it's not necessary to specify path to the config file